src/decisionengine/framework/dataspace/datasources/null.py

```diff
@@ -0,0 +1,62 @@
+import decisionengine.framework.dataspace.datasource as ds
+
+
+class NullSource(ds.DataSource):  # pragma: no cover
```

Comment:
Initially, I thought about suggesting NullDataSource as name, to distinguish more from Sources. But probably the different package is sufficient.

Reply:
Honestly, `NullDataSource` is a much better name.  I'll update the code.

Reply:
Updated